sql/2020/14_PWA/workbox_usage.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# Workbox usage - based on 2019/14_05.sql
+SELECT
+  client,
+  COUNT(DISTINCT page) AS freq,
+  total,
+  COUNT(DISTINCT page) / total AS pct
+FROM
+  `httparchive.almanac.service_workers`
+JOIN
+  (SELECT client, COUNT(DISTINCT page) AS total FROM `httparchive.almanac.service_workers` WHERE date = '2020-08-01' GROUP BY client)
+USING (client),
+  UNNEST(REGEXP_EXTRACT_ALL(body, r'new Workbox|new workbox|workbox\.precaching\.|workbox\.strategies\.')) AS occurrence
```

Comment:
Consider `(?i)` for case insensitivity, although the class name may be case sensitive, so feel free to close this.
```suggestion
  UNNEST(REGEXP_EXTRACT_ALL(body, r'(?i)new workbox|workbox\.precaching\.|workbox\.strategies\.')) AS occurrence
```

Reply:
Fixed (same results but better to be case insensitive).